### DISCUSSION TITLE
Not sure what is being asked for here.

### DISCUSSION THREAD

**Discussion by Yamanipanuchi** [guydavis/machinaris#365] [Q&A]
![image](https://user-images.githubusercontent.com/4724631/138782578-67d5bb49-33d6-4096-a9e9-545a4e8622e5.png)

I dont know what I am supposed to put here? I already tried it with what was already there and or corse got the error. So I just edit that line to what is needed and run composer again?

**Comment by Yamanipanuchi** [comment]
![image](https://user-images.githubusercontent.com/4724631/138946807-900866c2-6b44-4c88-8cb5-a02769538116.png)

I see that this is supposed to be a path, I am still unclear as to what the path should be? Name of the server?

**Reply by raingggg** [reply]
It seems that you are mounting whole host disks to the docker-container, which leads to the error.

The desired one should be mounting few host paths to docker-container paths. Let's say you have 3 disks in host machine:
/mnt/disk1
/mnt/disk2
/mnt/disk3

You wish map them into container with: 
/plots1
/plots2
/plots3

You configuration should looks like:
volumes:
- "/mnt/disks1/chives:/plots1" 
- "/mnt/disk2/chives:/plots2" 
- "/mnt/disk3/chives:/plots3" 

environment:
- plots_dir=/plots1:/plots2:/plots3 


Use https://www.machinaris.app/ to generate the mappings should be fine for most of the cases. Please try it.
![mount](https://user-images.githubusercontent.com/93122768/139171963-03430fd0-97ce-4e9a-9edc-c1d08214dd4e.png)

+/- button can be used to add more disks mappings
Used for dropdown can be used to choose whether it is final plots folder(for farming) or plotting folder(for plotting)
